int16 device_address
int8 tracking_status
int32 local_system_status
int32 local_filter_status
int32 data_valid_flags
int32 local_unix_time_stamp_seconds
int32 local_microseconds
float64 local_latitude
float64 local_longitude
float64 local_height
float32 local_velocity_north
float32 local_velocity_east
float32 local_velocity_down
float32 local_roll
float32 local_pitch
float32 local_heading
float32 local_latitude_standard_deviation
float32 local_longitude_standard_deviation
float32 local_height_standard_deviation
float32 local_roll_standard_deviation
float32 local_pitch_standard_deviation
float32 local_heading_standard_deviation
float32 local_depth
int32 remote_age_in_microseconds
float32 remote_range
float32 remote_azimuth
float32 remote_elevation
float32 remote_position_raw_x
float32 remote_position_raw_y
float32 remote_position_raw_z
float32 remote_position_x
float32 remote_position_y
float32 remote_position_z
float32 remote_north
float32 remote_east
float32 remote_down
float64 remote_latitude
float64 remote_longitude
float64 remote_height
float32 remote_range_standard_deviation
float32 remote_azimuth_standard_deviation
float32 remote_elevation_standard_deviation
float32 remote_latitude_standard_deviation
float32 remote_longitude_standard_deviation
float32 remote_height_standard_deviation
float32 remote_depth
